packages/SwingSet/src/controller/startXSnap.js

```diff
@@ -7,18 +7,21 @@ import { xsnap, recordXSnap } from '@agoric/xsnap';
 const NETSTRING_MAX_CHUNK_SIZE = 12_000_000;
 
 /**
- * @param {{ moduleFormat: string, source: string }[]} bundles
  * @param {{
- *   snapStore?: SnapStore,
+ *   bundleHandler: import('./bundle-handler.js').BundleHandler,
+ *   snapStore?: import('@agoric/swing-store').SnapStore,
  *   spawn: typeof import('child_process').spawn
  *   debug?: boolean,
  *   workerTraceRootPath?: string,
+ *   overrideBundles?: import('../types-external.js').Bundle[],
  * }} options
  */
-export function makeStartXSnap(bundles, options) {
+export function makeStartXSnap(options) {
   // our job is to simply curry some authorities and settings into the
   // 'startXSnap' function we return
+  const { bundleHandler } = options; // required unless bundleIDs is empty
   const { snapStore, spawn, debug = false, workerTraceRootPath } = options;
+  const { overrideBundles } = options;
```

Comment:
Suggestion: destructure `options` only once (but ordering subject to preference).
```suggestion
  // our job is to simply curry some authorities and settings into the
  // 'startXSnap' function we return
  const {
    snapStore,
    spawn,
    // required unless bundleIDs is empty
    bundleHandler,

    overrideBundles,
    debug = false,
    workerTraceRootPath,
  } = options;
```

Reply:
Yeah, I guess you're right. I was trying to discourage `prettier` from wrapping everything, 3 lines intead of 10, but it probably isn't worth it.